Sources/TSCBasic/misc.swift

```diff
@@ -110,9 +115,14 @@ public func lookupExecutablePath(
     }
 
     // Ensure the value is not a path.
-    if !value.contains("/") {
+    if !isPath {
         // Try to locate in search paths.
         paths.append(contentsOf: searchPaths.map({ $0.appending(component: value) }))
+#if os(Windows)
+        if !value.contains(".") {
```

Comment:
I think that we should get the last arc and ensure that does not have a `.`.  `\path\name.dir\name` should not be treated as having a `.` in it.

Reply:
This piece of code is wrapped inside `if !isPath`, while `!isPath` indicates that the input `value` is a single component to search for.